packages/components/src/Notification/Notification.scss

```diff
@@ -44,6 +44,11 @@ $icon-size: 16px;
 			color: $tc-notification-error-color;
 		}
 
+		svg {
+			height: $icon-size;
```

Comment:
You should set `$icon-size: $svg-md-size;` from [this file from Boostrap theme](https://github.com/Talend/ui/blob/master/packages/theme/src/theme/_guidelines.scss)